### PR TITLE
Added report markers with backend integration, updated dockerfile to migrate db on build

### PR DESCRIPTION
**Pull request by tabernik**
Basic disturbance report markers added. Integrated with backend, so refreshing page will keep markers. 

May need to make migrations once with "docker-compose exec backend python manage.py makemigrations" as I changed something in the crimeReport in the backend. 

Also updated dockerfile to migrate when building, but forgot to update README to remove migration line.

### CODE REVIEW

File: streetsense/docker-compose.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ services:
   backend:
     build:
       context: ./streetsense_backend
-    command: python manage.py runserver 0.0.0.0:8000
+    command: sh -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```
Comment (renakollmannsuhr) [inline streetsense/docker-compose.yml:17]: shouldn't migrate happen after the runserver command? I got errors when it was this way.

**Comment by renakollmannsuhr** [comment]
good news is location works! (it was disabled on my sys settings lol). But I got axios errors retrieving the reports and creating one 
Screenshot 2024-10-26 at 14 41 36

File: streetsense/docker-compose.yml (hunk 1 above)
Comment (tabernik) [reply streetsense/docker-compose.yml:17]: Have you ran makemigrations once with the docker containers running? I made a change to the CrimeReport table and I think django needs to generate a report every time that happens. I forgot to add the report file, so it is the 2nd commit to this push, if you pull again you should have it, but you might still need to make migrations. 

This made the report for me:

    docker-compose exec backend python manage.py makemigrations 

I tried switching the order of migrate and manage and it won't work for me.

Comment (renakollmannsuhr) [reply streetsense/docker-compose.yml:17]: yea I had the newest commit pulled. But it works now! idk why

Comment (renakollmannsuhr) [reply streetsense/docker-compose.yml:17]: I still get errors in the backend when running migrate before run-server though 
Screenshot 2024-10-26 at 14 55 28

Comment (renakollmannsuhr) [reply streetsense/docker-compose.yml:17]: I think running both commands at once might happen before the database is actually up and running

Comment (renakollmannsuhr) [reply streetsense/docker-compose.yml:17]: as the manage.py migrate command is only for us devs when we make a change to the database, can't we just run it manually instead?

Comment (tabernik) [reply streetsense/docker-compose.yml:17]: Yea that might be for the best.

**Review by renakollmannsuhr — APPROVED** [review]

**Pull request merged**